node/Cargo.toml

```diff
@@ -138,17 +138,13 @@ wip = [
     "common/wip",
     "dex-api-rpc/wip",
     "liquidity-proxy-rpc/wip",
+    "beefy-light-client",
+    "beefy-light-client-rpc",
```

Comment:
Why did you move it into `wip` if it should be released?

Reply:
Beefy is not part of Federated bridge